### DISCUSSION TITLE
Help with a zombie folder in my github pages web-page structure.

### DISCUSSION THREAD

**Discussion by ke0ff** [community/community#110763] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a static web-page which features several sub-folders.  I changed a couple of pdf files in one of those sub-folders and am unable to see the changes outside my repo browser.  In fact, none of the changes I make to files in that sub-folder (including index.html) are propagating.  I made similar change to pdf in another sub-folder, and that change has propagated to the open web.  I've reviewed the paths of others with similar issues, and am unable to get any traction with this particular sub-folder.  In fact, I have now deleted the folder completely, yet it still shows up in my web browser.  I have (repeatedly) cleared all history entries from my web browser, yet the contents of the folder are still accessible.  At this point, I have reached the end of my rope.  If I am doing something wrong, I have no earthly idea what it is nor how to fix it.  ACH!!!

The URL in question is: https://ke0ff.github.io/hm133/index.html

Most of the links on that page link to the hm133 sub-folder.  As of this moment, the "hm133" folder (and thus its contents) is deleted from my repo.

My modus operondi for page maintenance in the past has been: 1) modify a file, 2) commit the modified file to the pages repo, 3) wait a few minutes and the changes are published.  Something here is broken between the commit and the publish and I don't have access to the tools to fix it (there is no "publish" or "build" button that I can see on my github web portal, just "COMMIT").

Cheers

**Comment by hyper-ayoub** [comment]
response#1
delete the repos of your website and add it again 

and go to the page on setting repos and add  host it 
that all

**Comment by ke0ff** [comment]
If I delete the entire page, the main part of the site goes away (404 on my browser) but the subfolder is still accessible.  This seems like a problem that is deep in some server somewhere...  The folder just WILL...NOT...DIE.

**Comment by ke0ff** [comment]
Renaming the folder "fixed the problem"  However, the original folder is still out there.  It is not getting deleted.  https://ke0ff.github.io/hm133/ should not exist, but yet... I still see it.
